src/app/home/home.component.ts

```diff
@@ -38,19 +21,60 @@ export class HomeComponent implements OnInit {
     totalCountFormatted: string
   };
 
-  highlightCards: readonly HighlightCard[] = [
+  private currentTime = new Date();
+
+  private highlightCards: readonly HighlightCard[] = [
+    {
+      appearAt: 'asap',
+      disappearAt: CCOpenDate,
+      headerText: 'Christmas Challenge 2023 is starting soon!',
+      bodyText: 'Save the date 28th November - 5th December 2023',
+      iconColor: "brand-cc-red",
+      backgroundImageUrl: new URL('/assets/images/card-background-cc-lights.jpg', environment.donateGlobalUriPrefix),
+      button: {
+        text: 'Find out more',
+        href: new URL('/christmas-challenge-2023', environment.donateGlobalUriPrefix)
+      }
+    },
+    {
+      appearAt: CCOpenDate,
+      disappearAt: CCCloseDate,
+      headerText: 'Double the difference - Christmas Challenge',
+      bodyText: '28th November - 5th December 2023',
+      iconColor: "brand-cc-red",
+      backgroundImageUrl: new URL('/assets/images/card-background-cc-lights.jpg', environment.donateGlobalUriPrefix),
+      button: {
+        text: 'Donate now',
+        href: new URL('/christmas-challenge-2023', environment.donateGlobalUriPrefix)
+      }
+    },
     {
-      headerText: "Woman and Girls Match Fund",
-      bodyText: "11th - 18th October 2023",
-      iconColor: "brand-wgmf-purple",
-      backgroundImageUrl: new URL('/assets/images/wmg-purple-texture.jpg', environment.donateGlobalUriPrefix),
+      appearAt: CCCloseDate,
+      disappearAt: 'never',
+      headerText: 'Christmas Challenge 2023',
+      bodyText: '28th November - 5th December 2023',
+      iconColor: "brand-cc-red",
+      backgroundImageUrl: new URL('/assets/images/card-background-cc-lights.jpg', environment.donateGlobalUriPrefix),
       button: {
-        text: "See results",
-        href: new URL('/women-and-girls-2023/', environment.donateGlobalUriPrefix),
+        text: 'See Results',
+        href: new URL('/christmas-challenge-2023', environment.donateGlobalUriPrefix)
       }
     },
-    globalCard,
     {
+      appearAt: 'asap',
+      disappearAt: GMFApplicationCloseDate,
+      headerText: "Applications for Green Match Fund are now open!",
+      bodyText: "Apply by January 16 2024",
```

Comment:
Should we standardise on a date format for these? Some copy has the number with suffix followed by the month

Reply:
Yes we probably should standardise, either to have the suffix or not have it. I don't have a strong feeling which way, I think I'd rather we leave it out everywhere. @BethBigGive what do you think?

Reply:
Merged so will move question to ticket.